### PR TITLE
Mayhem Heroes Integration

### PR DESCRIPTION
**Pull request by capuanob**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Build Stage
```
Comment (xansec) [inline Dockerfile:1]: Please don't use git clone or `ldd`.

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+image: ghcr.io/capuanob/ufbx-fuzz:latest
```
Comment (xansec) [inline Mayhemfile:1]: Please omit image name. Tests -> testsuite. Any reason for the long `max_length`?

File: fuzz.c [hunk 3]
```diff
@@ -0,0 +1,9 @@
+#include <stdint.h> // uint8_t
```
Comment (xansec) [inline fuzz.c:1]: Can we rename this to indicate what's being fuzzed?

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: Mayhemfile (hunk 2 above)
Comment (capuanob) [reply Mayhemfile]: long max_length is to facilitate the large corpus seed, which is pigeonholed by the fact that 3D assets are large (even simple ones).

**Comment by capuanob** [comment]
@xansec Done!

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**